ext/console/02_console.js

```diff
@@ -1547,15 +1547,51 @@
     if (!colorEquals(css.backgroundColor, prevCss.backgroundColor)) {
       if (css.backgroundColor != null) {
         const [r, g, b] = css.backgroundColor;
-        ansi += `\x1b[48;2;${r};${g};${b}m`;
+        if (colorEquals(css.backgroundColor, [0, 0, 0])) {
```

Comment:
I don't agree with mapping the matching hex values to the 8 terminal colours, what we should do is move the `parseColor()` step over here and short-circuit at the colour keywords before calling it.

Reply:
Done!